FixWhitespace.hs

```diff
@@ -49,6 +49,7 @@ data Options = Options
   -- ^ The location to the configuration file.
   , optTabSize :: String
   -- ^ The number of spaces to expand a tab character to.  @"0"@ for keeping tabs.
+  , optConsEL  :: String
```

Comment:
For symmetry with the above, it'd be great to have a haddock for the new field.

Reply:
Yes, please.

Reply:
Please suggest the wording.

Reply:
Sorry, I'll get to this today, I hope

Reply:
Here's a suggestion:

```suggestion
  , optConsEL  :: String
  -- ^ The number of consecutive empty lines allowed. Unlimited if 0.
```

I also want to ask to be more generous about the field name length: "EL" looks too short. Other field names here don't use any shortcuts at all (unless we count Tab as a shortcut to Tabular :-)).

I'd propose to replace `optConsEL` with `optConsequtiveEmptyLines` (or `optRunningEmptyLines` is even better to my eye).

Reply:
it's consecutive, there is no q.

Should I align the `::`?

Reply:
And which, `optConsecutiveEmptyLines`, or `optRunningEmptyLines`; if latter, should the flag name be also changed?

Reply:
Pardon my grammar! Hopefully, you got my point. 

Yeah, I'd change it for "running" everywhere. But that's a mild preference.

Reply:
maybe `--running-empty-lines` would be better actually

Reply:
I committed the suggestion last week, is there still something?